### PR TITLE
intro screen ui

### PR DESCRIPTION
**Pull request by Andrew-Marchment**
- set up ui for intro screen. Might need a bit more tweaking but should be a good start.
- still need to hook everything up

### CODE REVIEW

File: app/components/PostsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          <ScrollArea className={`bg-card p-4 rounded-lg ${maxHeight}`}>
+            <div key={i}>
+              <h3 className="font-semibold">Tweet {i + 1}</h3>
+              <p>
+                This is an example tweet content. It can be much longer in real
+                scenarios.
+              </p>
+            </div>
+          </ScrollArea>
+        ))} */}
```
Comment (zaurbek-stark) [inline app/components/PostsScreen.tsx:50]: What is this code? do we need it?

File: components/ui/button.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import * as React from "react"
+import { Slot } from "@radix-ui/react-slot"
+import { cva, type VariantProps } from "class-variance-authority"
+
+import { cn } from "@/lib/utils"
+
+const buttonVariants = cva(
```
Comment (zaurbek-stark) [inline components/ui/button.tsx:7]: Insteresting, never used cva before. It that something useful? 
The code is a bit hard to read actually, does it give some advantages compared to using tailwindcss?

**Review by zaurbek-stark — APPROVED** [review]

File: app/components/PostsScreen.tsx (hunk 1 above)
Comment (Andrew-Marchment) [reply app/components/PostsScreen.tsx:50]: That was just some test stuff I forgot to delete. Will get rid of it.

File: components/ui/button.tsx (hunk 2 above)
Comment (Andrew-Marchment) [reply components/ui/button.tsx:7]: That's used in some shadcn components like this one. Makes it easy to pick between those pre-built styles and I think you can also add your own variants.

**Pull request merged**